src/plugin/pelias/geocoder/attrcard.js

```diff
@@ -1,48 +1,62 @@
-goog.provide('plugin.pelias.geocoder.AttrCardCtrl');
-goog.provide('plugin.pelias.geocoder.attrCardDirective');
+goog.module('plugin.pelias.geocoder.AttrCardUI');
 
-goog.require('os');
-goog.require('os.ui.Module');
+const {ROOT} = goog.require('os');
+const Module = goog.require('os.ui.Module');
+
+const AttrResult = goog.requireType('plugin.pelias.geocoder.AttrResult');
 
 
 /**
  * The geonames result card directive for displaying search results.
  *
  * @return {angular.Directive}
  */
-plugin.pelias.geocoder.attrCardDirective = function() {
-  return {
-    replace: true,
-    restrict: 'E',
-    templateUrl: os.ROOT + 'views/plugin/pelias/geocoder/attrcard.html',
-    controller: plugin.pelias.geocoder.AttrCardCtrl,
-    controllerAs: 'attrCtrl'
-  };
-};
+const directive = () => ({
+  replace: true,
+  restrict: 'E',
+  templateUrl: ROOT + 'views/plugin/pelias/geocoder/attrcard.html',
+  controller: Controller,
+  controllerAs: 'attrCtrl'
+});
+
+/**
+ * The element tag for the directive.
+ * @type {string}
+ */
+const directiveTag = 'peliasgeocoderattrcard';
 
 
 /**
  * Register the directive.
  */
-os.ui.Module.directive('peliasgeocoderattrcard', [plugin.pelias.geocoder.attrCardDirective]);
-
+Module.directive('peliasgeocoderattrcard', [directive]);
```

Comment:
Same directive tag comment as above. Might be something easy to tweak in the scripts.